packages/encoding/src/__tests__/index-test.ts

```diff
@@ -27,3 +32,37 @@ test("stringToBase64urlString", async function () {
     "eyJwdWJsaWNLZXkiOiIwMmY3MzlmOGM3N2IzMmY0ZDVmMTMyNjU4NjFmZWJkNzZlN2E5YzYxYTExNDBkMjk2YjhjMTYzMDI1MDg4NzAzMTYiLCJzaWduYXR1cmUiOiIzMDQ0MDIyMDJhOTJjMjRlNGI0ZGUzY2RiNWMwNWEyYjFmNDIyNjRiYTgxMzljZjY2YjJkMWVjZjBhMDk5ODdhYjlhMmZlY2IwMjIwM2JmZDkxZDhjNWU4N2Y3OGRhOGI1Y2Y1ZGRiMjdjOTZjYjAwYjg0ODc5N2QwZmM3M2JmMzcxODkyYzQyM2Y4MSIsInNjaGVtZSI6IlNJR05BVFVSRV9TQ0hFTUVfVEtfQVBJX1AyNTYifQ"
   );
 });
+
+// Convert base64String to base64UrlEncodedString
+test("base64StringToBase64UrlEncodedString", async function () {
+  expect(
+    base64StringToBase64UrlEncodedString("YW55IGNhcm5hbCBwbGVhc3VyZS4=")
+  ).toBe("YW55IGNhcm5hbCBwbGVhc3VyZS4");
+  expect(base64StringToBase64UrlEncodedString("aGVsbG8gd29ybGQ=")).toBe(
+    "aGVsbG8gd29ybGQ"
+  );
+  expect(base64StringToBase64UrlEncodedString("U29tZSBzYW1wbGUgdGV4dA==")).toBe(
```

Comment:
can remove this newline. or add a newline between 40 and 41 (just want to be consistent)